job-server/src/main/scala/spark/jobserver/WebApi.scala

```diff
@@ -181,6 +181,11 @@ class WebApi(system: ActorSystem,
       ServerSSLEngineProvider { engine =>
         val protocols = config.getStringList("spray.can.server.enabledProtocols")
         engine.setEnabledProtocols(protocols.toArray(Array[String]()))
+        val sprayConfig = config.getConfig("spray.can.server")
+        if(sprayConfig.hasPath("truststore")) {
```

Comment:
space before ( ...   we should enforce this via ScalaStyle I guess

Reply:
I think there's no point in that one line since the line above have the same issue.
I this it's best merging this and doing a global refactoring once the ScalaStyle rules are implemented.

Reply:
I guess that's fine.